patches/server/0019-Implement-Paper-VersionChecker.patch

```diff
@@ -109,6 +109,7 @@ index 0000000000000000000000000000000000000000..1a1b50e475b9ede544b2f6d0d36632b2
 +    private static int fetchDistanceFromGitHub(@Nonnull String repo, @Nonnull String branch, @Nonnull String hash) {
 +        try {
 +            HttpURLConnection connection = (HttpURLConnection) new URL("https://api.github.com/repos/" + repo + "/compare/" + branch + "..." + hash).openConnection();
```

Comment:
I'd rather we rewrite this to use Java 11's client which also provides a default agent.